scripts/Dockerfile-CentOS-6

```diff
@@ -56,6 +56,7 @@ RUN make install
 WORKDIR /src
 RUN curl -O https://www.openssl.org/source/openssl-1.1.0j.tar.gz
 RUN tar xf openssl-1.1.0j.tar.gz
+RUN mkdir -p /etc/ssl/certs/
```

Comment:
If we're adding it to the image, do we really also need to add this line in `.gitlab-ci.yml`?

Reply:
We can remove them from the .gitlab scripts after we regenerate all the docker images, but for now we need it in both.